pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -701,9 +701,12 @@ let
     } // optionalAttrs (stdenv.hostPlatform.system == "x86_64-linux" || stdenv.hostPlatform.system == "aarch64-linux") {
       # Enable memory hotplug support
       # Allows you to dynamically add & remove memory to a VM client running NixOS without requiring a reboot
+      ACPI_HOTPLUG_CPU = yes;
       ACPI_HOTPLUG_MEMORY = yes;
+      MEMORY_HOTPLUG_AUTOONLINE = yes;
```

Comment:
```
warning: unused option: MEMORY_HOTPLUG_AUTOONLINE
```

The configuration option is not part of the [LKDDb](https://cateee.net/lkddb/).

Reply:
I missed that. I'll try again without that option to confirm.